Towny/src/main/java/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -2088,31 +2097,35 @@ public void reloadConfig(CommandSender sender, boolean reset) {
 		}
 		
 		try {
-			TownySettings.loadConfig(Paths.get(TownyUniverse.getInstance().getRootFolder()).resolve("settings").resolve("config.yml"), plugin.getVersion());
+			plugin.loadConfig(true);
 			TownySettings.loadTownLevelConfig();   // TownLevel and NationLevels are not loaded in the config,
 			TownySettings.loadNationLevelConfig(); // but later so the config-migrator can do it's work on them if needed.
-			Translation.loadTranslationRegistry();
+			plugin.loadLocalization(true);
 			TownBlockTypeHandler.initialize();
-		} catch (TownyException e) {
+
+			TownyMessaging.sendMsg(sender, Translatable.of("msg_reloaded_config"));
+			plugin.removeError(TownyInitException.TownyError.MAIN_CONFIG);
+			plugin.removeError(TownyInitException.TownyError.LOCALIZATION);

```

Comment:
These two are already being called when we're in plugin#loadConfig and plugin#loadLocalization, do we need to call them again?

Reply:
My sources say no